CRM/Civixero/Contact.php

```diff
@@ -151,6 +151,7 @@ public function pull(array $params): void {
       // Since we expect this to wind up in the job log we'll print the errors
       throw new CRM_Core_Exception(E::ts('Not all records were saved') . ': ' . print_r($errors, TRUE), 'incomplete', $errors);
     }
+    return ['AccountContactIDs' => $ids ?? []];
```

Comment:
return array of AccountContactIDs

Reply:
Yeah I'm really not loving this - I've been trying to think of something that would look less odd

Reply:
I don't really mind what the return looks like, it's just *really* useful to have a list of what the API call touched when troubleshooting.